src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  font-size: 12px;
+  font-weight: 500;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  color: $nav-links-color;
```

Comment:
The variable `$nav-links-color` is used here, but ensure that it is defined elsewhere in your SCSS files. If not, this will cause a compilation error.